src/MapperContext.php

```diff
@@ -175,9 +175,9 @@ public static function withReference(array $context, string $reference, &$object
     /**
      * Check whether an attribute is allowed to be mapped.
      */
-    public static function isAllowedAttribute(array $context, string $attribute, $value): bool
+    public static function isAllowedAttribute(array $context, string $attribute, bool $valueIsNotNullOrNotUndefined): bool
```

Comment:
Does this method is internal or not ? 

If yes then this is a BC break and i will update the PR accordingly

Reply:
I would consider this internal, also we included AutoMapper version into Mapper hash, so if you update AutoMapper the Mapper will be re-generated so there won't be any issue there ~

Reply:
at some point, shouldn't all public methods here be marked internal?

Reply:
I will mark the class as internal then

Reply:
I mark the method as internal